pkg/cluster/backupstorage/s3.go

```diff
@@ -36,14 +36,11 @@ func NewS3Storage(s3Ctx s3config.S3Context, kubecli kubernetes.Interface, cluste
 			if err := os.MkdirAll(dir, 0700); err != nil {
 				return nil, err
 			}
-			credsFile, configFile, err := setupAWSConfig(kubecli, ns, p.S3.AWSSecret, dir)
+			options, err := setupAWSConfig(kubecli, ns, p.S3.AWSSecret, dir)
```

Comment:
Just to make sure. This is only changing the per cluster option, right? This is fine. But please mention it in "backup_config.md" per cluster option section.